iridium-core-server/src/test/java/software/iridium/api/instantiator/AccessTokenEntityInstantiatorTest.java

```diff
@@ -48,10 +52,13 @@ public void instantiate_AllGood_BehavesAsExpected() {
 
     when(mockTokenGenerator.generateAccessToken(same(identityId), any(Date.class)))
         .thenReturn(accessTokenValue);
+    when(mockRefreshTokenEntityInstantiator.instantiate(accessTokenValue))
+        .thenReturn(mockRefreshTokenEntity);
 
     final var response = subject.instantiate(identityId);
 
     verify(mockTokenGenerator).generateAccessToken(same(identityId), any(Date.class));
+    verify(mockRefreshTokenEntityInstantiator).instantiate(same(accessTokenValue));
 
     assertThat(response.getAccessToken(), is(equalTo(accessTokenValue)));
```

Comment:
Let's also assert that the refreshToken is properly associated with accessToken. 

e.g. 
```
    assertThat(response.getRefreshToken(), is(sameInstance(mockRefreshTokenEntity)));
```

Reply:
+1